chsdi/models/vector/bafu.py

```diff
@@ -2326,3 +2326,35 @@ class LandesForstInventarVegetationsHoehenModellRelief(Base, LandesForstInventar
     __bodId__ = 'ch.bafu.landesforstinventar-vegetationshoehenmodell_relief'
 
 register(LandesForstInventarVegetationsHoehenModellRelief.__bodId__, LandesForstInventarVegetationsHoehenModellRelief)
+
+
+class LandesforstinventarRegion:
+    __table_args__ = ({'schema': 'wald', 'autoload': False})
+    __template__ = 'templates/htmlpopup/landesforstinventar-region.mako'
+    __label__ = 'id'
+    id = Column('id', Integer, primary_key=True)
```

Comment:
id is part of the data delivery? Are you sure that it is stable?

Reply:
Yes, according to the DIF it is.